shortest_path/dijkstras.py

```diff
@@ -18,27 +17,25 @@ def make_graph():
 
 def dijkstras_heap(G, start='A'):
     shortest_paths = {} 
-    visited = {}
+    visited = set()
```

Comment:
This one is the change noted in the issue about representing the set as a dict when a set would do.